.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: alphagov/govuk-content-schemas
+          ref: deployed-to-production
+          path: govuk-content-schemas
```

Comment:
As I said on Slack I think we're going to find this tries to be linted as well. We may need to clone the govuk-developer-docs repo into a directory of it's own to isolate it. I guess either that or some env var or config manipulation, though path separation seems cleaner.

Reply:
I've attempted to clone it into `../govuk-content-schemas`, because then we don't need to set the envvar either as that's the default directory that the `govuk_schemas` gem (⁉️) looks in.

Reply:
Looks like it doesn't work like that:

```
 Clone govuk-content-schemas0s
##[error]Repository path '/home/runner/work/govuk-developer-docs/govuk-content-schemas' is not under '/home/runner/work/govuk-developer-docs/govuk-developer-docs'
Run actions/checkout@v2
  with:
    repository: alphagov/govuk-content-schemas
    ref: deployed-to-production
    path: ../govuk-content-schemas
    token: ***
    ssh-strict: true
    persist-credentials: true
    clean: true
    fetch-depth: 1
    lfs: false
    submodules: false
##[error]Repository path '/home/runner/work/govuk-developer-docs/govuk-content-schemas' is not under '/home/runner/work/govuk-developer-docs/govuk-developer-docs'
```

I suppose if Actions cleans up after itself, I guess it only cleans up the current runner directory...

Reply:
Heh - yeah I think that was quite optimistic. Would feel like a bit of a hack. Worth a try though.

We'll have to do similar with vendor/bundle but that'd feel a bit less odd to have the linting ignore directory.